app/uk/gov/hmrc/fileupload/ApplicationLoader.scala

```diff
@@ -35,23 +35,24 @@ import play.api.mvc.EssentialFilter
 import play.api.routing.Router
 import play.modules.reactivemongo.ReactiveMongoComponentImpl
 import reactivemongo.api.commands
-import uk.gov.hmrc.fileupload.admin.{Routes => AdminRoutes}
-import uk.gov.hmrc.fileupload.app.{Routes => AppRoutes}
+import uk.gov.hmrc.fileupload.admin.{Routes ⇒ AdminRoutes}
+import uk.gov.hmrc.fileupload.app.{Routes ⇒ AppRoutes}
```

Comment:
Are aware of this special character? It isn't bad but it is hard to type it from keyboard without help.

Reply:
got you thanks